koa-crud/src/graphql/type-defs/promos.ts

```diff
@@ -29,6 +29,17 @@ export default gql`
     INACTIVE
   }
 
+  type PromoConnectionEdge {
+    node: Promo
+    cursor: String
+  }
+
+  type PromoConnection {
+    totalCount: Int
+    edges: [PromoConnectionEdge]
```

Comment:
`edges` should be like this `[PromoConnectionEdge]!`

Reply:
noted sir